src/main/java/it/gov/pagopa/pu/migration/dto/debtpositiontypeorgoperator/DebtPositionTypeOrgOperatorMigrationFileDTO.java

```diff
@@ -0,0 +1,25 @@
+package it.gov.pagopa.pu.migration.dto.debtpositiontypeorgoperator;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import com.opencsv.bean.CsvBindByName;
+
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@SuperBuilder
+public class DebtPositionTypeOrgOperatorMigrationFileDTO {
+
+  @CsvBindByName(column = "org_ipa_code", required = true)
+  private String orgIpaCode;
+
+  @CsvBindByName(column = "cf_operator_hash", required = true)
+  private byte[] cfOperatorHash;
```

Comment:
from the CSV you cannot read it as ciphered! you will read the plain value, or the fiscal... you will next cipher it before to store on the db